### PR TITLE
Document ssl_verify: truststore

### PR DESCRIPTION
**Pull request by zeehio**
### Description

On  #13075 I submitted a contribution to use the operating system certificate store. It got merged some months ago.

While the introduced feature had tests and basic docs it did not appear in the user guide.

This pull request just aims to complete the previous pull request adding a paragraph to the user guide.

I think it is relevant because for many corporate users behind an SSL proxy the `ssl_verify: truststore` option will make configuration and certificate management easier.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests? **Does not apply**
- [x] Add / update outdated documentation? **Done**

### CODE REVIEW

File: docs/source/user-guide/configuration/settings.rst [hunk 1]
```diff
@@ -238,6 +238,18 @@ connection's normal security and is not recommended:
 
   ssl_verify: False
 
+.. versionadded:: 23.9.0
+   The ``ssl_verify: truststore`` setting is only available with conda 23.9.0 or later and using Python 3.10 or later.
+
+If the certificate authority is already trusted by the operating
+system, for instance because it was installed by a system
+administrator, you can tell conda to use the operating system
+certificate store.
```
Comment (travishathaway) [inline docs/source/user-guide/configuration/settings.rst:247]:
```suggestion
certificate store by setting ``ssl_verify`` to "truststore":
```

**Review by travishathaway — CHANGES_REQUESTED** [review]
Just a small change request. Otherwise looks good.

**Comment by zeehio** [comment]
Done, thanks for reviewing!

**Review by travishathaway — APPROVED** [review]
🚀

**Pull request merged**